### ISSUE TITLE
ansible-service-broker deployment is not idempotent

### ISSUE THREAD

**Issue by mcanevet** [openshift/openshift-ansible#6089]
#### Description

Using release-3.7 branch I get this error when running twice:

```
FAILED! => {"changed": false, "failed": true, "msg": {"cmd": "/usr/bin/oc replace -f /tmp/asb-Y5_Jkm -n openshift-ansible-service-broker", "results": {}, "returncode": 1, "stderr": "The Service \"asb\" is invalid: spec.clusterIP: Invalid value: \"\": field is immutable\n", "stdout": ""}}
```

##### Version

```
# ansible --version
ansible 2.3.1.0
  config file = /root/openshift-ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.13 (default, Jan 19 2017, 14:48:08) [GCC 6.3.0 20170118]

# git describe 
openshift-ansible-3.7.5-1
```

**Comment by pradeeppandeyy** [comment]
Same Issue

**Comment by vdbxavier** [comment]
Same Issue

**Comment by fabianvf** [comment]
should have been fixed by #6277
